scripts/scil_tractogram_resample.py

```diff
@@ -62,30 +60,29 @@ def _build_arg_parser():
 
     # For upsampling:
     upsampling_group = p.add_argument_group('Upsampling params')
-    std_group = upsampling_group.add_mutually_exclusive_group()
-    std_group.add_argument('--point_wise_std', type=float,
-                           help='Noise to add to existing streamlines\'' +
-                                ' points to generate new ones.')
-    std_group.add_argument('--streamline_wise_std', type=float,
-                           help='Noise to add to existing whole' +
-                                ' streamlines to generate new ones.')
-    sub_p = upsampling_group.add_mutually_exclusive_group()
-    sub_p.add_argument('--gaussian', metavar='SIGMA', type=int,
-                       help='Sigma for smoothing. Use the value of surronding'
-                            ' X,Y,Z points on \nthe streamline to blur the'
-                            ' streamlines. A good sigma choice would \nbe '
-                            'around 5.')
-    sub_p.add_argument('--spline', nargs=2, metavar=('SIGMA', 'NB_CTRL_POINT'),
-                       type=int,
-                       help='Sigma and number of points for smoothing. Models '
-                            'each streamline \nas a spline. A good sigma '
-                            'choice would be around 5 and control \npoints '
-                            'around 10.')
-
-    upsampling_group.add_argument(
-        '--keep_invalid_streamlines', action='store_true',
-        help='Keep invalid newly generated streamlines that may '
-             'go out of the \nbounding box.')
+    upsampling_group.add_argument('--point_wise_std', type=float, default=1,
+                                  help='Noise to add to existing streamlines '
+                                       'points to generate new ones [%(default)s].')
+    upsampling_group.add_argument('--tube_radius', type=float, default=1,
+                                  help='Maximum distance to generate streamlines '
+                                       ' around the original ones [%(default)s].')
+    upsampling_group.add_argument('--force_tube', action='store_true',
+                                  help='Force the use of parellel transport to '
+                                       'resample, even if the output tractogram '
+                                       'has fewer streamlines.')
```

Comment:
I'm not sure I understand this. Can you explain why parallel transport could result in fewer streamlines?

Also, in the description of tube_radius, above, you don't talk about parallel transport. So it sounds a little weird that --force_tube suddenly talks about it. I would expand the description of both args.